storage-transfer/package.json

```diff
@@ -12,9 +12,9 @@
     "node": ">=8"
   },
   "scripts": {
-    "unit-test": "repo-tools test run --cmd ava -- -T 20s --verbose test/*.test.js",
-    "system-test": "repo-tools test run --cmd ava -- -T 20s --verbose system-test/*.test.js",
-    "test": "npm run unit-test && npm run system-test"
+    "unit-test": "mocha test/*.test.js --timeout=10000",
```

Comment:
So no need to block on this feedback, but I would rather just use `mocha --timeout=10000` across the board here.  That way if a new test gets added to the `test` directory, you don't have to worry about it having the right prefix.

Reply:
Happy to change it, I'll do it in a separate PR for all the folders.